design/benchmark_tests.md

```diff
@@ -0,0 +1,30 @@
+# Benchmarks Tests
+
+## Requirements
+1. Measure the latency percentiles (50th, 90th, 95th, 99th 99.9) for a batch of repeated basic operations (e.g. 10K put requests)  
```

Comment:
1. Think we should address the concurrency level we perform these tests as some of the assumptions we made in the design and the implementation also build on the fact that we need design for throughput.
2. Like Nessie I think that running the benchmark tests should enable easy point and run on different environments - even locally if needed.

Reply:
1. Agreed. Added a clarification and an option to run it with different parallelism levels.
2. Added.